test/Confluent.Kafka.UnitTests/Serialization/ByteArray.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Confluent.Kafka.Serialization;
+using Xunit;
+
+namespace Confluent.Kafka.UnitTests.Serialization
```

Comment:
We have a mix of Confluent.Kafka.Tests and Confluent.Kafka.UnitTests namespace in UnitTests project, didn't mention it yet. We should put all on Confuent.Kafka.UnitTests